### DISCUSSION TITLE
--chemistry选择问题

### DISCUSSION THREAD

**Discussion by li1311139481** [singleron-RD/CeleScope#282] [Q&A]
我得到了一个文库，尝试--chemistry auto发生了报错
`Valid chemistry read counts percent < 0.1`
尝试--chemistry scopeV1没有发生报错，但是我得到的结果仅仅只有99个cell barcode
我找到了关于chemistry的定义，然后与我的文库进行对比
```
PATTERN_DICT = {
    'auto': None,
    'scopeV1': 'C12U8T18',
    'scopeV2.0.0': 'C8L16C8L16C8U8T18',
    'scopeV2.0.1': 'C8L16C8L16C8L1U8T18',
    'scopeV2.1.0': 'C8L16C8L16C8U12T18',
    'scopeV2.1.1': 'C8L16C8L16C8L1U12T18',
    'scopeV2.2.1': 'C8L16C8L16C8L1U12T18',
    'scopeV3.0.1': 'C9L16C9L16C9L1U12T18',
    # flv_rna is actually U9L16, but the last 10 bases can not be sequenced accurately.
    'flv_rna': 'C8L16C8L16C8U9L6',
    'flv' : 'U9C8L16C8L16C8',
    'bulk_vdj': 'L18C6U16',
    'bulk_rna': 'C9U12',
    'customized': None,
}
```
我的文库结构如下
![image](https://github.com/singleron-RD/CeleScope/assets/109973822/e45b6b8a-bd69-4187-a0e9-a18e60aaeea7)
我发现我的pattern不符合任何一个文库的pattern，我该如何指定--chemistry？

**Comment by zhouyiqi91** [comment]
这个文库来自于BD
https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&acc=SRR23250411&display=metadata
可以用BD的软件分析。

**Reply by li1311139481** [reply]
谢谢您的帮助，确实是BD的文库
